site/content/docs/5.3/helpers/colored-links.md

```diff
@@ -16,14 +23,20 @@ You can use the `.link-*` classes to colorize links. Unlike the [`.text-*` class
 **Heads up!** `.link-body-emphasis` is currently the only colored link that adapts to color modes. It's treated as a special case until v6 arrives and we can more thoroughly rebuild our theme colors for color modes. Until then, it's a unique, high-contrast link color with custom `:hover` and `:focus` styles. However, it still responds to the new link utilities.
 {{< /callout >}}
 
-<!--Boosted mod: feature limited to primary and light, we don't loop over "theme-colors"; body-emphasis is treated on its own-->
+<!--Boosted mod: use `contrast_color` as background ensure a good contrast-->
 {{< example >}}
-<p><a href="#" class="link-primary">Primary link</a></p>
-<p><a href="#" class="link-light">Light link</a></p>
+{{< colored-links.inline >}}
+  {{- range (index $.Site.Data "theme-colors") }}
+    <p><a href="#" class="link-{{ .name }}{{ with .contrast_color }} bg-{{ . }}{{ end }}">{{ .name | title }} link</a></p>
```

Comment:
There's an issue with the indentation in the documentation
![Screenshot 2023-07-11 at 09 33 09](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/assets/17381666/d3aef7d0-6480-4fa2-a98b-a60d7bd88e3d)